airflow/executors/executor_type_aliases.py

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
We can probably remove `executor` from the filename as we have it in the path.

Reply:
I removed this file altogether and moved all the types closer to where they are defined.